myapp/app/controllers/articles_controller.rb

```diff
@@ -11,6 +11,10 @@ def new
 		@article = Article.new
 	end
 
+	def edit
+		@article = Article.find(params[:id])
```

Comment:
Here you could use memorization. Whats memorization?

```ruby

def edit
   article
end

private

def article
   @article ||= Article.find(params[:id])
end

def update
   if article.update(article_params)
   ....
end
```

So the first time you call `article`, `@article` will have your value, and if you call it again, it will just use it